tests/unit/App.spec.ts

```diff
@@ -1970,6 +1970,7 @@ describe('App as a COMPLETED Registration Application', () => {
   it('fetches NR data properly', () => {
     expect(rootStore.getNameRequest.nrNum).toBe('NR 1234567')
     expect(businessStore.getLegalName).toBe('My Name Request')
+    expect(businessStore.getOperatingName).toBe('My Name Request')
```

Comment:
This check is good as it verifies that operating name was populated correctly when this SP registration is loaded, but it would be even better if the operating name was different from the legal name.

Testing this may be a bit tricky. Basically, I think you'll have to update the (operating) name in the NR above (lines 1895+).

I believe this is where the legal name is set for a draft app: https://github.com/bcgov/business-filings-ui/blob/3479b284afd2232bcab6243ce408d83d714967fb/src/App.vue#L654

In this special case, it appears we don't have a separate legal name (in the business object). But this is a special case so I'm OK with that. (We plan to update this special case later anyway.)